### PR TITLE
More column improvements

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR:
  * Updates dependencies on `base` and `core-java` employing recently introduced `DefaultRepository`, `withId()` API.
  * Updates the code to use `VBuilder` for creating messages.
  * Migrates test to Truth.
  * Simplifies Hello World example eliminating unnecessary classes.
  * Reworks Kanban example (see details below) removing work tracking features and introducing Column Aggregate.

### Kanban example changes
 * Work status features were removed. We may return them if we see that it fits the purpose of the example and associated tutorial. The application is already reach enough for demoing working with aggregates and process managers in concert. We may not have features related to showing tuples (which were in the previous version). But tuples can be shown in a dedicated example.
 * `ColumnAggregate` was introduced so that we can show orchestration of aggregates by a process manager. We also may want to add command for adding columns.
 * The board is now initialized with a set of default columns, which is handled by `BoardInitProcess`.
 * Moving a card from column to column is also a process (see `MoveCardProcess`).

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@armiol, PTAL.

File: kanban-board/src/main/java/io/spine/examples/kanban/event/CardMovingEvent.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import io.spine.examples.kanban.ColumnId;
+
+import java.util.Set;
+import java.util.function.Supplier;
+
+import static io.spine.server.route.EventRoute.noTargets;
+import static io.spine.server.route.EventRoute.withId;
+
+/**
+ * Common interface for events participating in moving a card form column to column.
```
Comment (armiol) [inline kanban-board/src/main/java/io/spine/examples/kanban/event/CardMovingEvent.java]: “from”

File: kanban-board/src/main/java/io/spine/examples/kanban/server/board/BoardAggregate.java [hunk 2]
```diff
@@ -23,120 +23,76 @@
...
-                .build();
+    @Apply
+    void event(ColumnAdded e) {
+        builder().addColumn(e.getColumn());
     }
 
+    /**
+     * Whenever a card created, it is placed to the first column of the board.
+     *
+     * @implNote This board knows its column. So the board listens to the events on new card
```
Comment (armiol) [inline kanban-board/src/main/java/io/spine/examples/kanban/server/board/BoardAggregate.java]: I assume you meant “its columns”.

File: kanban-board/src/main/proto/spine_examples/kanban/commands.proto [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+// A command to create a new card on the specified board.
+//
+// New cards are placed into the first column of the board.
+//
+message CreateCard {
+
+    // The ID for newly created card.
+    CardId card = 1;
+
+    // Example note:
```
Comment (armiol) [inline kanban-board/src/main/proto/spine_examples/kanban/commands.proto]: This comment section looks misplaced and a bit misformatted.

**Pull request merged**